### PR TITLE
Preparing release 0.5.0

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,20 +15,20 @@ def read_file(filename):
...
+    "cliquet[monitoring,postgresql]>=3.1.4,<3.2",
     "kinto>=2.1,<2.2",
     "kinto-attachment>=0.5.1,<0.6",
     "kinto-amo>=0.1.1,<0.2",
     "kinto-changes>=0.2,<0.3",
-    "kinto-signer>=0.4,<0.5",
+    "kinto-signer>=0.5,<0.6",
     "cliquet-fxa>=1.4,<1.5",
-    "boto>=2.39,<2.40",
+    "boto>=2.40,<2.41",
```
Comment (leplatrem) [inline setup.py]: Are we sure this works with kinto-attachment ?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
 
+- **Cliquet 3.1.4**: https://github.com/mozilla-services/cliquet/releases/tag/3.1.4
+- **kinto 2.1.1**: https://github.com/Kinto/kinto/releases/tag/2.1.1
+- **kinto-attachment 0.5.1**: https://github.com/Kinto/kinto-attachment/releases/tag/0.5.1
 - **kinto-amo 0.1.1**: https://github.com/mozilla-services/kinto-amo/releases/tag/0.1.1
-- **kinto-signer 0.4.0**: https://github.com/Kinto/kinto-signer/releases/tag/0.4.0
+- **kinto-changes 0.2.0**: https://github.com/Kinto/kinto-changes/releases/tag/0.2.0
+- **kinto-signer 0.5.0**: https://github.com/Kinto/kinto-signer/releases/tag/0.5.0
+- **cliquet-fxa 1.4.0**: https://github.com/mozilla-services/cliquet-fxa/releases/tag/1.4.0
+- **boto 2.40**: https://github.com/boto/boto/releases/tag/2.40.0
```
Comment (leplatrem) [inline CHANGELOG.rst]: Changelog is here

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
 ==================
 
 **Version control**
 
+- **Cliquet 3.1.4**: https://github.com/mozilla-services/cliquet/releases/tag/3.1.4
+- **kinto 2.1.1**: https://github.com/Kinto/kinto/releases/tag/2.1.1
+- **kinto-attachment 0.5.1**: https://github.com/Kinto/kinto-attachment/releases/tag/0.5.1
 - **kinto-amo 0.1.1**: https://github.com/mozilla-services/kinto-amo/releases/tag/0.1.1
-- **kinto-signer 0.4.0**: https://github.com/Kinto/kinto-signer/releases/tag/0.4.0
+- **kinto-changes 0.2.0**: https://github.com/Kinto/kinto-changes/releases/tag/0.2.0
```
Comment (leplatrem) [inline CHANGELOG.rst]: Do you repeat what didn't change ?

File: setup.py (hunk 1 above)
Comment (Natim) [inline setup.py]: According to https://travis-ci.org/Kinto/kinto-attachment/builds/129995597 yes

File: CHANGELOG.rst (hunk 3 above)
Comment (Natim) [inline CHANGELOG.rst]: Yes it makes it easier to read.

**Comment by leplatrem** [comment]
r+

**Pull request merged**